src/Types.jl

```diff
@@ -455,7 +460,12 @@ function stdlibs()
     end
     return STDLIB[]
 end
-is_stdlib(uuid::UUID) = uuid in keys(stdlibs())
+is_stdlib(uuid::UUID) = uuid in keys(stdlibs()) && uuid ∉ UPGRADABLE_STDLIBS_UUIDS
+# Includes upgradable stdlibs
+function is_any_stdlib(uuid::UUID, julia_version::Union{VersionNumber, Nothing})
+    return is_stdlib(uuid, julia_version) || uuid in UPGRADABLE_STDLIBS_UUIDS
+end
```

Comment:
I renamed and tweaked some things slightly. I don't really think that test is needed since it is true "by construction" and the test would be written virtually identically to the implementation.